features/main/always_identifier.feature

```diff
@@ -0,0 +1,117 @@
+Feature: AlwaysIdentifier support
+  In order to use a hypermedia API
```

Comment:
You can create a new context method in `FeatureContext` like https://github.com/api-platform/core/blob/master/features/bootstrap/FeatureContext.php#L125 to create new `AlwaysIdentifiersDummies`, remove the create and delete scenarios, and move the scenarios for each format (JSON-LD, HAL, JSON) in the corresponding directories (we prefer to group tests per format instead of per feature, it will help when we will create a subtree split).